crypto/cipher/aes256cbc/decrypter.go

```diff
@@ -40,7 +40,7 @@ type Decrypter struct {
 func (d Decrypter) Decrypt(data mc.EncryptedContent) (mc.PlainContent, error) {
 	encryptedData, err := bytesDecode(data)
 	if err != nil {
-		return nil, errors.WithMessage(err, "could not convert encryptedData")
+		return nil, mc.ErrDecrypt()
 	}
 
 	return decryptEncryptedData(d.privateKey, encryptedData)
```

Comment:
`decryptEncryptedData(d.privateKey, encryptedData)` also returns errors they should be `mc.ErrDecrypt`